src/components/login/Login.js

```diff
@@ -8,18 +8,24 @@ import {Link, useNavigate} from 'react-router-dom'
 function Login() {
     const [email, setEmail] = useState('');
     const [password, setPassword] = useState('')
+    const [error, setError] = useState(false)
     const dispatch = useDispatch()
     const navigate = useNavigate();
 
     async function signIn(e) {
         e.preventDefault()
-        const res = await axios({method:'post', url:'http://localhost:5000/login', data: {username: email, password: password},  headers: {"Content-Type": "multipart/form-data"}})
-        if(res.status = 200){
-            dispatch({
-                type: set_user,
-                user: res.data.username
-            })
-            navigate(-1)
+        try {
+            const res = await axios({method:'post', url:'http://localhost:5000/login', data: {username: email, password: password},  headers: {"Content-Type": "multipart/form-data"}})
+            if(res.status = 200){
+                dispatch({
+                    type: set_user,
+                    user: res.data.username
```

Comment:
Let's change this from username to email to be consistent with the front end and desired end user experience.

Reply:
Changed username to email